packages/broker/test/integration/plugins/operator/OperatorValueBreachWatcher.test.ts

```diff
@@ -0,0 +1,128 @@
+const theGraphUrl = `http://${STREAMR_DOCKER_DEV_HOST}:8800/subgraphs/name/streamr-dev/network-subgraphs`
+
+const logger = new Logger(module)
+
+const STREAM_CREATION_KEY = '0xb1abdb742d3924a45b0a54f780f0f21b9d9283b231a0a0b35ce5e455fa5375e7'
+
+async function getTotalUnwithdrawnEarnings(operatorContract: Operator): Promise<bigint> {
+    const { earnings } = await operatorContract.getEarningsFromSponsorships()
+    let unwithdrawnEarnings = BigInt(0)
+    for (const e of earnings) {
+        unwithdrawnEarnings += e.toBigInt()
+    }
+    logger.debug(`Total unwithdrawn earnings: ${formatEther(unwithdrawnEarnings.toString())} (t = ${Date.now()})`)
+    return unwithdrawnEarnings
+}
+
+describe('OperatorValueBreachWatcher', () => {
+    let provider: Provider
+    let adminWallet: Wallet
+    let token: TestToken
+    let streamId: string
+
+    const deployNewOperator = async () => {
+        const operatorWallet = await generateWalletWithGasAndTokens(provider, fastChainConfig)
+        logger.debug('Deploying operator contract')
+        const operatorContract = await deployOperatorContract(fastChainConfig, operatorWallet, { operatorSharePercent: 10 })
+        logger.debug(`Operator deployed at ${operatorContract.address}`)
+        const operatorConfig = {
+            operatorContractAddress: toEthereumAddress(operatorContract.address),
+            provider,
+            theGraphUrl,
+            signer: operatorWallet,
+            maxSponsorshipsCount: 20,
+            minSponsorshipEarnings: 1 // full tokens
+        }
+        return { operatorWallet, operatorContract, operatorConfig }
+    }
+
+    beforeAll(async () => {
+        provider = new JsonRpcProvider(`http://${STREAMR_DOCKER_DEV_HOST}:8547`)
+        logger.debug('Connected to: ', await provider.getNetwork())
+
+        adminWallet = new Wallet(STREAM_CREATION_KEY, provider)
+        const streamRegistry = new Contract(fastChainConfig.contracts.StreamRegistry, streamRegistryABI, adminWallet) as unknown as StreamRegistry
+        logger.debug('Creating stream for the test')
+        const createStreamReceipt = await (await streamRegistry.createStream(
+            `/operatorvaluewatchertest-${Date.now()}`,
+            '{"partitions":1}')
+        ).wait()
+        streamId = createStreamReceipt.events?.find((e) => e.event === 'StreamCreated')?.args?.id
+        const streamExists = await streamRegistry.exists(streamId)
+        logger.debug('Stream created:', { streamId, streamExists })
+        // TODO: use createClient once configs allow it. For now I'm creating the stream directly using the contract
+        // const client = createClient(STREAM_CREATION_KEY)
+        // streamId = (await createTestStream(client, module)).id
+        // await client.destroy()
+
+        token = new Contract(fastChainConfig.contracts.DATA, tokenABI) as unknown as TestToken
+    }, 60 * 1000)
+
+    it('can find a random operator, excluding himself', async () => {
```

Comment:
This is really testing the method `getRandomOperator` of class `MaintainOperatorValueHelper`.

I would create a new integration test `MaintainOperatorValueHelper.test.ts" and move this test there.

Reply:
I noticed this wasn't moved. I would move it as done here https://github.com/streamr-dev/network/pull/1763

Reply:
Sorry about this one. I see now I missed it..